archibishop/github-connect
# #158787482 Improve gradle build speeds

#### What does this PR do?
Improves grade build speeds 
#### Description of Task to be completed?
Improves grade build speeds by enabling grade build cache
#### How should this be manually tested?
- Clone the repository.
- Build the project
- Rebuild the project.
- You should notice a drop in build times for project
#### What are the relevant pivotal tracker stories?
[#158787482](https://www.pivotaltracker.com/story/show/158787482)
#### Screenshots (if appropriate)
Before
<img width="1366" alt="screen shot 2018-10-01 at 10 45 52" src="https://user-images.githubusercontent.com/39958516/46285428-14caf380-c584-11e8-8fc9-f70eaa96e9c9.png">
After
<img width="1368" alt="screen shot 2018-10-01 at 10 51 55" src="https://user-images.githubusercontent.com/39958516/46285433-1c8a9800-c584-11e8-9234-06ec79039f91.png">
